src/Handler/Workspace/Drop/DropWorkspaceHandler.php

```diff
@@ -57,38 +75,59 @@ public function __invoke(
         $keyData = json_decode($command->getWorkspaceUserName(), true, 512, JSON_THROW_ON_ERROR);
 
         $cloudResourceManager = $this->clientManager->getCloudResourceManager($credentials);
-        $getIamPolicyRequest = new GetIamPolicyRequest();
-        $projectCredentials = CredentialsHelper::getCredentialsArray($credentials);
-        $projectName = 'projects/' . $projectCredentials['project_id'];
-        $actualPolicy = $cloudResourceManager->projects->getIamPolicy($projectName, $getIamPolicyRequest);
-        $actualBinding[] = $actualPolicy->getBindings();
-
-        $newBinding = [];
-        /** @var Binding $binding */
-        foreach ($actualBinding[0] as $binding) {
-            $tmpBinding = new Binding();
-            $tmpBinding->setRole($binding->getRole());
-            if ($binding->getCondition() !== null) {
-                $tmpBinding->setCondition($binding->getCondition());
+
+        $setIamPolicyRetryPolicy = new CallableRetryPolicy(function (Throwable $e) {
+            if (in_array($e->getCode(), array_merge(self::ERROR_CODES_FOR_RETRY, [409]))) {
```

Comment:
409 - davam to cele do retry v pripade, ze by tam nejak vznikol konflikt nech sa nacita cely ten biding znova a spravi sa to cele znova

Reply:
Luks gut akorát bych si to hodil do konstant oboje ten merdž je divný. Normálně bych použil spread toto by mělo byt validní.
```
    private const ERROR_CODES_FOR_RETRY = [401, 403, 429];
    private const ERROR_CODES_FOR_RETRY_IAM = [409, ...self::ERROR_CODES_FOR_RETRY];
```

Reply:
Jasne diky, upravil som to, testy mi lokalne presli tak to davam na review